ci-scripts/unit-tests.sh

```diff
@@ -2,7 +2,7 @@
 set -e
 set -o pipefail
 
-EXCLUDE_APPLICATIONS=storefrontapp
+EXCLUDE_APPLICATIONS=storefrontapp,ssr-tests
```

Comment:
`query:` Do we want to disable `ssr-tests` tests by default? Or is it for when the CI is ready?

Reply:
I think this will prevent causing issues with unit tests as-is since ci will try to include these ssr-tests. They require their own mechanism. 

But I think the README.md is a great idea!